benchmarks/fadbench.c

```diff
@@ -36,17 +33,19 @@ inline static void *Execute(void *Arg) {
         for (j = 0; j < rnum; j++)
             ;
     }
+    BarrierWait(&bar);
+    if (id == 0) d2 = getTimeMillis();
+
     return NULL;
 }
 
 int main(int argc, char *argv[]) {
     parseArguments(&bench_args, argc, argv);
-    SHARED_OBJECT_INIT();
+    object = 1;
```

Comment:
Food for thought:
A good practice is to differentiate the local vs global vars by using different name conventions e.g.:
int localVar;
int GlobalVar;
that makes the code easier to follow, review.

Reply:
Maybe in a next PR, we start adopting a new naming-scheme for variables, at least for the benchmarks.
Please open an issue for this.